.azure-pipelines/build-template.yml

```diff
@@ -82,20 +84,31 @@ jobs:
       CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=${{ parameters.cache_mode }} SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/${{ parameters.platform }}"
       ENABLE_DOCKER_BASE_PULL=y make configure PLATFORM=${{ parameters.platform }} PLATFORM_ARCH=${{ parameters.platform_arch }}
       trap "sudo rm -rf fsroot" EXIT
-      if [ ${{ parameters.dbg_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) INSTALL_DEBUG_TOOLS=y target/sonic-${{ parameters.platform }}.bin && \
-          mv target/sonic-${{ parameters.platform }}.bin target/sonic-${{ parameters.platform }}-dbg.bin
-      fi
-      if [ ${{ parameters.swi_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) ENABLE_IMAGE_SIGNATURE=y target/sonic-aboot-${{ parameters.platform }}.swi
-      fi
-      if [ ${{ parameters.raw_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) target/sonic-${{ parameters.platform }}.raw
-      fi
-      if [ ${{ parameters.sync_rpc_image }} == true ];then
-        make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) ENABLE_SYNCD_RPC=y target/docker-syncd-${{ parameters.platform_short }}-rpc.gz
+
+      if [ ${{ parameters.platform }} == vs ]; then
+        if [ ${{ parameters.dbg_image }} == true ]; then
+          make USERNAME=admin $CACHE_OPTIONS SONIC_BUILD_JOBS=$(nproc) INSTALL_DEBUG_TOOLS=y target/sonic-vs.img.gz && \
```

Comment:
> && \ [](start = 115, length = 5)

Remove them and just use 2 lines?

Reply:
only when build successfull will do the mv

Reply:
there is `set -e`

Reply:
will look at it later.